.travis.yml

```diff
@@ -12,7 +12,7 @@ after_failure: R -q -e 'tic::after_failure()'
 before_deploy: R -q -e 'tic::before_deploy()'
 deploy:
   provider: script
-  script: R -q -e 'tic::deploy()'
+  script: R -q -e 'remotes::install_github("mlr-org/mlr3pkgdowntemplate"); tic::deploy()'
```

Comment:
the travis yaml should not be touched according to the "tic philosophy". This goes in line with the approach that all custom steps should be specified in `tic.R`. (We'll hopefully soon have an (automated) way to update the yaml templates).

We handle pkgdown templates as follows:

https://github.com/ropenscilabs/tic/blob/755f76801dba360e76c15be2b6f2f6bcb0f45450/tic.R#L1-L2